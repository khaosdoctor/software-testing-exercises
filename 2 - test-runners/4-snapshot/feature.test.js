const { getUser } = require('./feature')

describe('Happy path', () => {
  it('should return the user with the given id from snapshot', async () => {
    const user = await getUser(1)

    expect(user).toMatchSnapshot()
  })

  it('should return the user with the given id from inline snapshot', async () => {
    const user = await getUser(2)

    expect(user).toMatchInlineSnapshot(`
{
  "age": 18,
  "id": 2,
  "name": "Jane",
}
`)
  })
})
