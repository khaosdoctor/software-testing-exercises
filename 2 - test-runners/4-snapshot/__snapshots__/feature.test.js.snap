// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Happy path should return the user with the given id from snapshot 1`] = `
{
  "age": 20,
  "id": 1,
  "name": "John",
}
`;
